tools/cli/util.go

```diff
@@ -75,7 +76,7 @@ func GetHistory(ctx context.Context, workflowClient sdkclient.Client, workflowID
 // HistoryEventToString convert HistoryEvent to string
 func HistoryEventToString(e *historypb.HistoryEvent, printFully bool, maxFieldLength int) string {
 	data := getEventAttributes(e)
-	return stringify.AnyToString(data, printFully, maxFieldLength, dataconverter.GetCurrent())
+	return stringify.AnyToString(data, printFully, maxFieldLength, customDataConverter())
```

Comment:
I think `dataconverter.GetCurrent()` already covers the case where user provided their own data converter by calling `SetCurrent` in `temporal/tools/cli/dataconverter/dataconverter.go`. So if that is not overridden by the user, then `GetCurrent()` returns the default data converter. Otherwise, it returns the user data converter.

Reply:
This is meant to be a no-op, just improving clarity of which converter is used where.